src/Compilers/CSharp/Test/Semantic/Semantics/InterpolationTests.cs

```diff
@@ -421,6 +421,25 @@ static void Main(string[] args)
                 );
         }
 
+        [Fact]
+        public void InvalidCharInFormatSpecifier()
+        {
+            string source =
+@"using System;
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine( $""{3:{}"" );
+    }
+}";
+            CreateCompilationWithMscorlib45(source).VerifyDiagnostics(
+                // (6,33): error CS1056: Unerwartetes Zeichen "{".
```

Comment:
Nit: should this be English to be consistent? 

(Although I quite like the idea of having different languages in the code base 😊)